packages/react-native/React/CoreModules/RCTAppearance.mm

```diff
@@ -80,6 +67,15 @@ @implementation RCTAppearance {
   NSString *_currentColorScheme;
 }
 
+- (instancetype)init
+{
+  if ((self = [super init])) {
+    UITraitCollection *traitCollection = RCTSharedApplication().delegate.window.traitCollection;
+    _currentColorScheme = RCTColorSchemePreference(traitCollection);
+  }
+  return self;
```

Comment:
Perhaps we should set-up the observer here by default (now in `startObserving`). Since otherwise `_currentColorScheme` will go out of date if there's no listeners on the JS side.